src/components/BookingForm.vue

```diff
@@ -31,26 +37,38 @@
 
 <script>
 import BadContainedButton from "@/components/BadContainedButton";
+import BadTextInput from "@/components/BadTextInput";
+
 export default {
   name: "BookingForm",
   data() {
     return {
       officeId: "",
-      bookingdate: "",
-      emailaddress: ""
+      bookingDate: "",
+      emailAddress: ""
     };
   },
   methods: {
+    changeOfficeId(value) {
```

Comment:
Why we assign the value on change event?
You can just set the v-model in the component and the value will be updated automatically.

Reply:
I need to somehow to tie to `value` prop of v-text-field through BadInput. I'll check if i can just provide v-model directly.

Reply:
removed `change` in favor of v-model